Resources/Prototypes/Body/Parts/human.yml

```
@@ -20,7 +20,7 @@
   parent: [PartHuman, BaseTorso]
   components:
   - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)